src/app.js

```diff
@@ -36,11 +30,9 @@ app.use(
 async function fetchAndTransformArticle(articleId, lang, accessToken) {
   const article = await fetchArticle(articleId, accessToken);
 
-  const rawContent = contentI18N(article, lang, true);
-  const footNotes = footNotesI18N(article, lang, true);
-  const rawIntroduction = introductionI18N(article, lang, true);
-
-  const introduction = rawIntroduction ? rawIntroduction.introduction : '';
+  const rawContent = article.content;
+  const footNotes = defined(article.footNotes, {});
+  const introduction = defined(article.introduction, '');
 
   const content = await transformContentAndExtractCopyrightInfo(
     rawContent,
```

Comment:
Bare send in `article.content` her istedenfor å assigne det til en egen variabel.